docs/en/latest/practices/how-to-use-go-plugin-runner-in-apisix-ingress.md

```diff
@@ -0,0 +1,347 @@
+
+How you choose the option should depend on your local hardware considerations. The reason for selecting the second option here is that I want to rely on my powerful local hardware to increase the building speed and speed up the process.
+
+### Go to the go-plugin-runner directory
+
+Choose a folder address `/home/chever/api7/cloud_native/tasks/plugin-runner` and place our `apisix-go-plugin-runner` project in this folder.
+
+After successful placement, the file tree is shown below:
+
+```bash
+chever@cloud-native-01:~/api7/cloud_native/tasks/plugin-runner$ tree -L 1
+.
+└── apisix-go-plugin-runner
+
+1 directory, 0 files
+```
+
+Then you need to go to the `apisix-go-plugin-runner/cmd/go-runner/plugins` directory and write the plugins you need in that directory. This article will use the default plugin `say` for demonstration purposes.
+
+```bash
+chever@cloud-native-01:~/api7/cloud_native/tasks/plugin-runner/apisix-go-plugin-runner$ tree cmd
+cmd
+└── go-runner
+    ├── main.go
+    ├── main_test.go
+    ├── plugins
+    │   ├── fault_injection.go
+    │   ├── fault_injection_test.go
+    │   ├── limit_req.go
+    │   ├── limit_req_test.go
+    │   ├── say.go
+    │   └── say_test.go
+    └── version.go
+
+2 directories, 10 files
+```
+
+After writing the plugins, start compiling the executable formally, and note here that you should build static executables, not dynamic ones.
+
+The package compile command is as follows.
+
+```bash
+CGO_ENABLED=0 go build -a -ldflags '-extldflags "-static"' .
+```
+
+This successfully packages a statically compiled `go-runner` executable.
+
+In the `apisix-go-plugin-runner/cmd/go-runner/` directory, you can see that the current file tree looks like this:
+
+```bash
+chever@cloud-native-01:~/api7/cloud_native/tasks/plugin-runner/apisix-go-plugin-runner/cmd/go-runner$ tree -L 1
+.
+├── go-runner
+├── main.go
+├── main_test.go
+├── plugins
+└── version.go
+
+1 directory, 4 files
+```
+
+Please remember the path `apisix-go-plugin-runner/cmd/go-runner/go-runner`, we will use it later.
+
+### Build Docker Image
+
+The image is built here in preparation for installing APISIX later using `helm`.
+
+#### Write Dockerfile
+
+Return to the path `/home/chever/api7/cloud_native/tasks/plugin-runner` and create a Dockerfile in that directory, a demonstration of which is given here.
+
+We can write the Dockerfile by referring to examples in this [repository](https://github.com/apache/apisix-docker). Here I recommend adding the following line of code to this [Dockerfile file](https://github.com/apache/apisix-docker/blob/master/alpine/Dockerfile).
+
+```bash
+ARG ENABLE_PROXY=false
+
+# Build Apache APISIX
+FROM api7/apisix-base:1.19.9.1.5
+
+# That is the line being added
+ADD ./apisix-go-plugin-runner /usr/local/apisix-go-plugin-runner
+
+ARG APISIX_VERSION=2.13.1
+LABEL apisix_version="${APISIX_VERSION}"
+
+ARG ENABLE_PROXY
+
+```
```

Comment:
```suggestion
```Dockerfile
FROM apache/apisix:2.13.1

COPY <Your go-plugin-runner static binary> /usr/local/apisix-go-plugin-runner
```
```

we can just base on APISIX image, then copy go-plugin-runner static binary